composer.json

```
@@ -6,10 +6,10 @@
 	"homepage": "https://github.com/M2mobi/ApnsPHP",
 	"license": "BSD-3-Clause",
 	"require": {
-		"php": ">=7.2.0",
+		"php": ">=7.4||>=8.0",
```

```suggestion
		"php": ">=7.4",
```

`>=7.4.0` already let people install it on PHP 8 or any version bigger or equal as 7.4.0